File: linear_search/java/linear_search.java
class MyLinearSearch {
 
    public static int linerSearch(int[] arr, int key){
         
        int size = arr.length;
        for(int i=0;i<size;i++){
            if(arr[i] == key){
                return i;
            }
        }
        return -1;
    }
     
    public static void main(String a[]){
         
        int[] arr1= {23,45,21,55,234,1,34,90};
        int searchKey = 34;
        System.out.println("Key "+searchKey+" found at index: "+linerSearch(arr1, searchKey));
        int[] arr2= {123,445,421,595,2134,41,304,190};
        searchKey = 421;
        System.out.println("Key "+searchKey+" found at index: "+linerSearch(arr2, searchKey));
    }
}

File: searching/linear_search/swift/linear_search.swift

func linearSearch(arr: [Int], key: Int) -> Int {
    for(idx, element) in arr.enumerated() {
        if (element == key) {
            return idx
        }
    }
    return -1
}

// Could you use this way for readability
//func linearSearch(arr: [Int], key: Int) -> Int {
//    return arr.enumerated().filter { $1 == key }.first?.offset ?? -1
//}

let arr1 = [23,45,21,55,234,1,34,90]
let searchKey = 21
print("Key \(searchKey) found at index: \(linearSearch(arr: arr1, key: searchKey))")